Yosemite/Yosemite/Stores/Order/OrdersUpsertUseCase.swift

```diff
@@ -72,10 +72,6 @@ struct OrdersUpsertUseCase {
         let siteID = readOnlyOrder.siteID
         let orderID = readOnlyOrder.orderID
 
-        guard readOnlyOrder.items.count > 0 else {
```

Comment:
Do you think it's worth adding a unit test for this case? Or given that we're just removing a condition, they aren't necessary?

Reply:
Good thought. I wasn't going to add one since it's just removing a condition, but I realized there aren't any unit tests that cover removing order items from storage. I went ahead and added one in [9c7bfad](https://github.com/woocommerce/woocommerce-ios/pull/13665/commits/9c7bfad0dd9477cfd926292002458a069e2ffd11).